account_avatax_exemption/models/exemption.py

```diff
@@ -0,0 +1,332 @@
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError, ValidationError
+
+
+class ExemptionRule(models.Model):
+    _name = "exemption.code.rule"
+    _description = "Avatax Custom Rules"
+
+    name = fields.Char(index=True, default=lambda self: self.env._("New"))
```

Comment:
```suggestion
    name = fields.Char(index=True, default=lambda self: _("New"))
```
We should not translate this value. so please restore it

Reply:
is the _ fn something different in this context?

Reply:
putting _ here is translating or what?
@dnplkndll 
@xaviedoanhduy 
Do you need to translate or not translate?

Reply:
> replace the call to translate a string (`_`) with `self.env._` for getting some performance improvement in some cases. See https://github.com/odoo/odoo/pull/174844.

only in some cases, also not needed. I checked the codebase and found that most of them will not translate this part